test/src/main/scala/scala/test/twitter_scrooge/BUILD

```diff
@@ -0,0 +1,100 @@
+load("//scala:scala.bzl",   "scala_binary", "scala_library")
+load("//twitter_scrooge:twitter_scrooge.bzl", "scrooge_scala_library")
+
+scrooge_scala_library(
+  name = "thrift1",
+  thrift_library = "//test/src/main/scala/scala/test/twitter_scrooge/thrift",
+  visibility = ["//visibility:public"],
+)
+
+scrooge_scala_library(
```

Comment:
Do we care which pass it was compiled in? It sounds like we just need to keep track of which files were already compiled, and ignore them (as they'll already be on the classpath). Shouldn't keeping a set of already compiled thrift paths be sufficient, then?

So in this case, we don't care if it compiles scroogeA or scroogeB first... if it compiles scroogeA first, it'll compile/code gen thriftA/thriftB at once. If it does scroogeB first, it'll gen/compile scrooge B and then when it gets to scrooge A, it'll know to omit scrooge B.